CoastalResilienceNetwork/GeositeFramework
# Integrate map_util widget into framework core

## Overview

Fixes various UX issues with the map_utils plugin by integrating it into the
framework core.

Changes:

* Replace custom dropdown with Bootstrap widget
* Merge CSS, templates, and locale files
* Simplify map_utils commands

### Demo

![image](https://cloud.githubusercontent.com/assets/43062/21119899/b0d47c5e-c092-11e6-9bd8-ac4655bf26f4.png)

### Notes

We may want to create a follow up design task to update the Bootstrap dropdown widget style.

## Testing Instructions

* Verify that each map_utils command works as expected
* Verify that sidebar plugins remain open and active while executing map_utils commands

Connects #732
Connects #734
Connects #738
